packages/wdio-browserstack-service/src/util.ts

```diff
@@ -67,3 +80,325 @@ export function getParentSuiteName(fullTitle: string, testSuiteTitle: string): s
+        }
+    }
+    // CircleCI
+    if (env.CI === 'true' && env.CIRCLECI === 'true') {
+        return {
+            name: 'CircleCI',
+            build_url: env.CIRCLE_BUILD_URL,
+            job_name: env.CIRCLE_JOB,
+            build_number: env.CIRCLE_BUILD_NUM
+        }
+    }
+    // Travis CI
+    if (env.CI === 'true' && env.TRAVIS === 'true') {
+        return {
+            name: 'Travis CI',
+            build_url: env.TRAVIS_BUILD_WEB_URL,
+            job_name: env.TRAVIS_JOB_NAME,
+            build_number: env.TRAVIS_BUILD_NUMBER
+        }
+    }
+    // Codeship
+    if (env.CI === 'true' && env.CI_NAME === 'codeship') {
+        return {
+            name: 'Codeship',
+            build_url: null,
+            job_name: null,
+            build_number: null
+        }
+    }
+    // Bitbucket
+    if (env.BITBUCKET_BRANCH && env.BITBUCKET_COMMIT) {
+        return {
+            name: 'Bitbucket',
+            build_url: env.BITBUCKET_GIT_HTTP_ORIGIN,
+            job_name: null,
+            build_number: env.BITBUCKET_BUILD_NUMBER
+        }
+    }
+    // Drone
+    if (env.CI === 'true' && env.DRONE === 'true') {
+        return {
+            name: 'Drone',
+            build_url: env.DRONE_BUILD_LINK,
+            job_name: null,
+            build_number: env.DRONE_BUILD_NUMBER
+        }
+    }
+    // Semaphore
+    if (env.CI === 'true' && env.SEMAPHORE === 'true') {
+        return {
+            name: 'Semaphore',
+            build_url: env.SEMAPHORE_ORGANIZATION_URL,
+            job_name: env.SEMAPHORE_JOB_NAME,
+            build_number: env.SEMAPHORE_JOB_ID
+        }
+    }
+    // GitLab
+    if (env.CI === 'true' && env.GITLAB_CI === 'true') {
+        return {
+            name: 'GitLab',
+            build_url: env.CI_JOB_URL,
+            job_name: env.CI_JOB_NAME,
+            build_number: env.CI_JOB_ID
+        }
+    }
+    // Buildkite
+    if (env.CI === 'true' && env.BUILDKITE === 'true') {
+        return {
+            name: 'Buildkite',
+            build_url: env.BUILDKITE_BUILD_URL,
+            job_name: env.BUILDKITE_LABEL || env.BUILDKITE_PIPELINE_NAME,
+            build_number: env.BUILDKITE_BUILD_NUMBER
+        }
+    }
+    // Visual Studio Team Services
+    if (env.TF_BUILD === 'True') {
+        return {
+            name: 'Visual Studio Team Services',
+            build_url: `${env.SYSTEM_TEAMFOUNDATIONSERVERURI}${env.SYSTEM_TEAMPROJECTID}`,
+            job_name: env.SYSTEM_DEFINITIONID,
+            build_number: env.BUILD_BUILDID
+        }
+    }
+    // if no matches, return null
+    return null
+}
+
+export async function getGitMetaData () {
+    var info: any = gitRepoInfo()
+    if (!info.commonGitDir) return {}
+    const { remote } = await pGitconfig(info.commonGitDir)
+    const remotes = Object.keys(remote).map(remoteName =>  ({ name: remoteName, url: remote[remoteName]['url'] }))
+    return {
+        'name': 'git',
```

Comment:
No need to have `'` around property names.
```suggestion
        name: 'git',
        ...
```